misc/bazel/cargo-gazelle/Cargo.toml

```
@@ -0,0 +1,40 @@
+[package]
+name = "cargo-gazelle"
+version = "0.1.0"
+edition.workspace = true
+rust-version.workspace = true
+publish = false
+
+[lints]
+workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

```suggestion
[package]
name = "cargo-gazelle"
description = "BUILD file generator for Bazel projects."
version = "0.0.0"
edition.workspace = true
rust-version.workspace = true
publish = false
authors = ["Materialize, Inc."]

[lints]
workspace = true

```

(I usually copy/paste from an existing Cargo.toml rather than using Cargo.new; avoids some cruft while also getting Materialize-specific idioms, like `version = "0.0.0"`.)

---

Ahh yeah I usually run `cargo new`, just copying from an existing crate is much better. Updated!